crates/fuel-core/src/graphql_api/api_service.rs

```diff
@@ -221,17 +221,23 @@ where
         .extension(ViewExtension::new())
         .finish();
 
-    let router = Router::new()
+    let unthrottled_routes = Router::new()
+        .route("/v1/health", get(health))
+        .route("/health", get(health));
+
+    let throttled_routes = Router::new()
         .route("/v1/playground", get(graphql_playground))
         .route("/v1/graphql", post(graphql_handler).options(ok))
         .route(
             "/v1/graphql-sub",
             post(graphql_subscription_handler).options(ok),
         )
         .route("/v1/metrics", get(metrics))
-        .route("/v1/health", get(health))
-        .route("/health", get(health))
-        .layer(ConcurrencyLimitLayer::new(concurrency_limit))
+        .layer(ConcurrencyLimitLayer::new(concurrency_limit));
```

Comment:
I think we want something like. Because graph-sub also is limited by number of subscriptions, and playground is one time query that returns pre rendered html

<img width="1270" alt="image" src="https://github.com/user-attachments/assets/9fe3bf85-f0de-44b3-901b-7c401123f63e">

Reply:
> Throteling metrics doesn't allow request data=)
are you saying that because metrics are throttled, it affects how graphql and graphql-sub are throttled as well? 

why shouldn't we throttle the metrics?

Reply:
Because if we have a lot of queries, it becomes impossible for us to track the state of the node from the Grafana=)

Reply:
okay, making the change

Reply:
addressed in 1a66422487